src/client/public/styles/dash.css

```diff
@@ -131,8 +133,25 @@ a {
 }
 
 /* Hide the timeline line after the last event  */
-.pr > .pr-header:last-child, .pr > .pr-event:last-child > .pr-event__bullet svg {
-  background: white;
+.pr, .pr > .pr-event:last-child > .pr-event__bullet svg {
+  background-color: var(--page-bg-color);
+}
+
+@keyframes flashKeyframe {
+  0% {
+    background-color: var(--actionable-highlight-color);
+  }
+  25% {
+    background-color: var(--actionable-highlight-color);
+  }
+  to {
+    background-color: var(--page-bg-color);
+  }
+}
+
+.pr.actionable-flash, .pr.actionable-flash > .pr-event:last-child > .pr-event__bullet svg {
+  animation-duration: 700ms;
+  animation-name: flashKeyframe;
```

Comment:
Just add a `animation-delay` here. You may also need `animation-fill-mode: backwards` as well.